docs/_docs/installation/macos.md

```diff
@@ -3,155 +3,28 @@ title: Jekyll on macOS
-Ruby versions. This is very useful when you need to be able to run a given Ruby version on a project.
-
-```sh
-# Install Homebrew
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-
-# Install rbenv and ruby-build
-brew install rbenv
-
-# Set up rbenv integration with your shell
-rbenv init
-
-# Check your installation
-curl -fsSL https://github.com/rbenv/rbenv-installer/raw/main/bin/rbenv-doctor | bash
-```
-
-Restart your terminal to apply your changes.
-Next, you can install the Ruby version you want. Let's install the latest stable version:
+To install Jekyll on macOS, you need a proper Ruby development environment. While macOS comes preinstalled with Ruby, we don't recommend using that version to install Jekyll. This external article goes over the various reasons [why you shouldn't use the system Ruby](https://www.moncefbelyamani.com/why-you-shouldn-t-use-the-system-ruby-to-install-gems-on-a-mac/).
 
-```sh
-rbenv install {{ site.data.ruby.current_version }}
-rbenv global {{ site.data.ruby.current_version }}
-ruby -v
-{{ site.data.ruby.current_version_output }}
-```
+Instead, you'll need to install a separate and newer version of Ruby using a version manager such as [asdf](https://asdf-vm.com/), [chruby](https://github.com/postmodern/chruby), [rbenv](https://github.com/rbenv/rbenv), or [rvm](https://rvm.io/). Version managers allow you to easily install multiple versions of Ruby, and switch between them.
 
-That's it! Head over to [rbenv command references](https://github.com/rbenv/rbenv#command-reference) to learn how to use different versions of Ruby in your projects.
+We recommend `chruby` because it's the most reliable and easiest to use. Follow this external guide for [installing Jekyll on a Mac](https://www.moncefbelyamani.com/how-to-install-jekyll-on-a-mac-the-easy-way/) and setting up a proper Ruby environment with `chruby`.
 
 ## Install Jekyll
 
-After installing Ruby, install Jekyll and Bundler.
-
-### Local Install
-
-Install the bundler and jekyll gems:
-
-```sh
-gem install --user-install bundler jekyll
-```
-
-Get your Ruby version:
-
-```sh
-ruby -v
-{{ site.data.ruby.current_version_output }}
-```
-
-Append your path file with the following, replacing the `X.X` with the first two digits of your Ruby version:
-
-```bash
-# If you're using Zsh
-echo 'export PATH="$HOME/.gem/ruby/X.X.0/bin:$PATH"' >> ~/.zshrc
-
-# If you're using Bash
-echo 'export PATH="$HOME/.gem/ruby/X.X.0/bin:$PATH"' >> ~/.bash_profile
-
-# Unsure which shell you are using? Type
-echo $SHELL
-```
-
-Check that `GEM PATHS:` points to your home directory:
-
-```sh
-gem env
-```
-
-{: .note .info}
-Every time you update Ruby to a version in which the first two digits change, update your path to match.
-
-### Global Install
-
-{: .note .warning}
-We recommend not installing Ruby gems globally to avoid file permissions problems and using `sudo`.
```

Comment:
File permissions problems are only an issue when trying to install gems using the system Ruby. When installing Ruby with a reliable version manager like `chruby`, `sudo` is never needed, and there aren't any file permissions problems.

Since we are discouraging using the system Ruby, this section is no longer needed.